helm_deploy/templates/deployment.yaml

```diff
@@ -191,6 +191,16 @@ spec:
               value: 'true'
             - name: ENABLE_PROMETHEUS_EXPORTER
               value: 'true'
+            - name: METABASE_DASHBOARD_ID
```

Comment:
It is possible that we could have more than one dashboard. Either that or we need to try several charts/tables in one dashboard. 
Do you get a separate secret for each dashboard? Or do we get one and then need a list of IDs?

Reply:
I'm making it so that we use a list of IDs and render them one by one.